scripts/issue/554/upgrade_bveOXD_LP.py

```diff
@@ -38,7 +38,9 @@ def main(simulation="false"):
     for attr in strat_contract.signatures:
         try:
             assert attributes[attr] == getattr(strat_contract, attr).call()
-        except:
+        # Should fail if assertion is wrong
+        except KeyError:
```

Comment:
catching only this exception for me locally seems to jump out in other type of exception `AssertionError`.

maybe should be instead `except (AssertionError, KeyError) as e:` or just check for *attr* which are in the *attributes* variable

Reply:
Interesting, I didn't get any assertion errors. The idea was for assertion errors to jump out, they would mean that either a view variable's value changed and this change should be accounted or that a new variable was introduced. In this case, neither of these happened and I don't see why this would be the case. 

Do you know which attribute caused the assertion error?

Reply:
for the record was **balanceOfRewards** discussed in dm